modules/ROOT/pages/salesforce/salesforce-connector-design-center.adoc

```diff
@@ -0,0 +1,130 @@
+= Salesforce - Design Center
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configure in Design Center
+
+. In Anypoint Platform > *Design Center*, click *Create* > *Mule Application*.
+. In the first Mule Application card, if you want a Salesfore event to initiate access to your application (that is, to trigger your application), choose one of the Salesforce operations. Alternatively, you can use an *HTTP Listener* or *Scheduler* trigger.
++
+Salesforce operations:
++
+* *On Deleted Object* - Delete an object.
+* *On Modified Object* - Modify an object.
+* *On New Object* - Create a new object. Possible object types are Contact, Lead, Account, Contracts, Opportunity, and custom objects. The object types correspond to the Salesforce object you want to create, modify, or delete.
+* *Replay Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place in the specified channel trigger the flow.
++
+NOTE: A channel or topic must be published to Salesforce before a subscription to the channel can be created.
++
+* *Replay Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place on records in the specified topic trigger the flow.
+* *Subscribe Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events that take place after you subscribe.
+* *Subscribe Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events that take place after you subscribe.
+
+== New Object Trigger
+
+The following shows how to configure the Salesforce New Object trigger.
+
```

Comment:
"Configure the Salesforce New Object trigger (why or to do what?)." You don't need the self-referential text in the intro to a task.